### DISCUSSION TITLE
Show "Extreme Power Saver"/"Power Saver" status of dGPU and any programs keeping GPU awake

### DISCUSSION THREAD

**Discussion by CNordy** [seerge/g-helper#331] [Ideas]
It would be great to see whether my dGPU was in "power saving" or "extreme power saving" modes and whether any programs are keeping the GPU awake.

Since yesterday, I noticed my dGPU was always on and kept not showing stats in Armoury Crate. I troubleshot the issue with MyAsus scans and reddit forums, but eventually uninstalled Armoury Crate and tried G-Helper. I was hoping this would solve my issue, but it's hard to tell whether G-Helper is showing me the temperature of my iGPU or dGPU as the temperature is still higher than it normally would be. I'm not sure what else changed since yesterday as nothing was installed that would keep the either GPU awake, but my temps are constantly 60C rather than 40-50C for CPU with dGPU off.

Running a G14 GA402RJ

**Comment by seerge** [comment]
@CNordy 
- Extreme Power saving is how armoury calls dGPU disabled (g-helper just honestly says that it's iGPU only in this case)
- Power saving - is controlled by windows itself / drivers, not by AC or g-helper.

G-helper gets gpu temperature from asus own sensor (the one that bios uses to control fans) :)
